src/scipp/experimental/plotting/wrappers.py

```diff
@@ -36,15 +49,66 @@ def plot(obj: Union[VariableLike, Dict[str, VariableLike]]) -> Figure:
         - dict of Variables
         - dict of DataArrays
         - dict of numpy ndarrays
+    aspect:
+        Aspect ratio for the axes.
+    cbar:
+        Show colorbar in 2d plots if `True`.
+    crop:
+        Set the axis limits. Limits should be given as a dict with one entry per
+        dimension to be cropped. Each entry should be a nested dict containing scalar
+        values for `'min'` and/or `'max'`. Example:
+        `da.plot(crop={'time': {'min': 2 * sc.Unit('s'), 'max': 40 * sc.Unit('s')}})`
+    errorbars:
+        Show errorbars in 1d plots if `True`.
+    grid:
+        Show grid if `True`.
+    mask_color:
+        Color of masks in 1d plots.
+    norm:
+        Set to 'log' for a logarithmic y-axis (1d plots) or logarithmic colorscale
+        (2d plots).
+    scale:
+        Change axis scaling between `log` and `linear`. For example, specify
+        `scale={'tof': 'log'}` if you want log-scale for the `tof` dimension.
+    title:
+        The figure title.
+    vmin:
+        Lower bound for data to be displayed (y-axis for 1d plots, colorscale for
+        2d plots).
+    vmax:
+        Upper bound for data to be displayed (y-axis for 1d plots, colorscale for
+        2d plots).
+    **kwargs:
+        All other kwargs are directly forwarded to Matplotlib, the underlying plotting
+        library. The underlying functions called are the following:
+        - 1d data with a non bin-edge coordinate: `plot`
+        - 1d data with a bin-edge coordinate: `step`
+        - 2d data: `pcolormesh`
 
     Returns
     -------
     :
         A figure.
     """
+    all_args = {
+        **{
+            'aspect': aspect,
+            'cbar': cbar,
+            'crop': crop,
+            'errorbars': errorbars,
+            'grid': grid,
+            'mask_color': mask_color,
+            'norm': norm,
+            'scale': scale,
+            'title': title,
+            'vmin': vmin,
+            'vmax': vmax
```

Comment:
Seems easy to mess up when we add new args?  Can we/should we use `inspect`?

Reply:
In fact I messed it up myself when adding `crop` ;-)

Reply:
done